### ISSUE TITLE
'prune_low_magnitude' of 'keras.layers.convolutional.Conv2D' returns None

### ISSUE THREAD

**Issue by ShravanKarthik94** [tensorflow/model-optimization#163]
When trying to prune the Cov2D layer in a custom network the below error is observed. 
**_TypeError: 'NoneType' object is not callable_**

Below is a snippet of the code:

from tensorflow_model_optimization.sparsity import keras as sparsity
from keras.layers.convolutional import Conv2D, Conv2DTranspose

pruning_prm = {'pruning_schedule': sparsity.PolynomialDecay(initial_sparsity=0, final_sparsity=0.5,begin_step=15000, end_step=75000, frequency=100)}
x = Conv2D(filters=n_filters, kernel_size=(kernel_size, kernel_size), kernel_initializer="he_normal",
                  padding="same")
x = sparsity.prune_low_magnitude(x, **pruning_prm) --> returns None

Information on versions of different modules:
Keras 2.3.0
tensorflow-estimator (2.0.0)
tensorflow-gpu (2.0.0)
tensorflow-hub (0.6.0)
tensorflow-model-optimization (0.1.3)
tensorflow-probability (0.8.0rc0)
tensorflow-serving-api-gpu (1.14.0)

Any pointers/leads is much appreciated.

**Comment by alanchiao** [comment]
@ShravanKarthik94: the pruning API only works with tf.keras (under TensorFlow) and not the Keras library [here](https://github.com/keras-team/keras). If you're interested in using the library, please try out Keras in TensorFlow. 

https://github.com/tensorflow/model-optimization/issues/166 will make this clearer in the future. 

Does that address your issue?

**Comment by ShravanKarthik94** [comment]
@alanchiao Thank you. Yes, I did convert my model to use tf.keras layers and it works without any issues. Thanks again for your help.

**Comment by bothrasumit** [comment]
> @alanchiao Thank you. Yes, I did convert my model to use tf.keras layers and it works without any issues. Thanks again for your help.

Hi @ShravanKarthik94 , can you please share how did you changed the model into an instance of tf.keras.Model. It would be great if you can share some sample code for the same.
Thank you.

**Comment by ShravanKarthik94** [comment]
This was my original keras model (for u-net)
https://github.com/MADHAVAN001/semantic-segmentation/blob/2255129dc5ca1c93dada871d448359333d77f625/models/unet.py

This is after converting tf.keras with prune_low_magnitude implemented.
https://github.com/MADHAVAN001/semantic-segmentation/blob/534c8b0333fb574ab4677da30f441bf94e353e22/models/unet_tf.py
